docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -96,7 +96,14 @@ version              | version number of the new release
 
 ## 6. Verify that MANIFEST is up to date and that tarball can be built
 
-> This section is not relevant for Zonemaster-GUI.
+> This section is *NOT* relevant for Zonemaster-GUI.
+
+> `share/Makefile` in Zonemaster-Engine and Zonemaster-CLI, respectively,
+> only works with GNU Make. FreeBSD Make is not "GNU Make". `gmake` in
+> FreeBSD (and seperately installed) is "GNU Make".
+
+    gmake -C share touch-po all # Extra step in "Engine" in FreeBSD only.
+    gmake -C share all          # Extra step in "CLI" in FreeBSD only.
```

Comment:
Why not just `gmake all` for all components except GUI on BSD?

Reply:
It does not work and I am not sure it is a good idea.

Main Makefile is generated from Makefile.PL and in the FreeBSD case probably adapted to FreeBSD Make. In FreeBSD it is probably safer to run "make" than "gmake".

And running "gmake" on the main Makefile would not even help because it contains "MAKE=make" for the execution of share/Makefile. Then you would need to run "make MAKE=gmake", but then that "MAKE" might be used for something where gmake is not suitable.

Reply:
My goodness. I'll look into making the Makefiles portable.